cookiecutter-django-ida/{{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/settings/utils.py

```diff
@@ -14,10 +13,8 @@ def get_env_setting(setting):
         error_msg = "Set the [%s] env variable!" % setting
         raise ImproperlyConfigured(error_msg)
 
-def get_logger_config(log_dir='/var/tmp',
-                      logging_env="no_env",
-                      edx_filename="edx.log",
-                      dev_env=False,
```

Comment:
While this parameter was unused in the body, it is still referenced in callers (see base.py, local.py).

Reply:
If you restore it, I can take care of either removing or fixing it in another PR.